project-reports/2024/2024-Q4-Hyperledger-Identus.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: 2024 Q3 Hyperledger Identus
+parent: 2024
+grand_parent: Project Updates
+nav_exclude: true
+---
+
+Created by Javier Ribó with input from other maintainers
+
+
+# Project Health
+
+The project is still new to the Hyperledger Ecosystem and continues to have steady growth of contributors.
+The project can now be found on the LFX dashboard which can now be checked for health insights. In the last quarter we have seen an increase on the number of github issues created while the active contributors have gone down 8%.
```

Comment:
can you link the [LFX Dashboard](https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/overview/github?project=identus&routedFrom=Github&bestPractice=false&repository=all&dateFilters=Last%20Quarter&dateRange=2024-07-01%20to%202024-09-30&compare=PP&granularity=week&hideBots=true)

Reply:
Link is now added u can solve the conversation!